plugins/modules/python_distribution.py

```diff
@@ -126,6 +145,11 @@ def main():
             else:
                 desired_attributes["content_guard"] = None
 
+        if remote_name is not None:
+            remote = PulpPythonRemote(module, {"name": remote_name})
+            remote.find(failsafe=False)
+            desired_attributes["remote"] = remote.href
```

Comment:
Is it possible to remove the remote again from an existing distribution?
In the case of content guards (a few lines above), the extra if was there to determine whether `""` was provided to remove a configured guard.

Reply:
Yes the remote can be removed.

Reply:
I'll change it accordingly, as soon as I'm in the office tomorrow.

Reply:
I amended my commit, adding the possibility to remove the remote and the necessary tests.